### PR TITLE
Pin mapping changes

### PR DESCRIPTION
**Pull request by marnovandermaas**
Mostly took Alex's changes for this. I tested this using Sonata software and it looks reasonable to me. @AlexJones0 feel free to change things if you think I am missing something.

### CODE REVIEW

File: sdk/boards/sonata-prerelease.json [hunk 1]
```diff
@@ -52,37 +76,29 @@
...
-        "spi0": {
+        "spi_board": {
             "start" : 0x80300000,
             "end"   : 0x80301000
         },
-        "spi1": {
+        "spi_lcd": {
             "start" : 0x80301000,
             "end"   : 0x80302000
         },
```
Comment (HU90m) [inline sdk/boards/sonata-prerelease.json:86]:
```suggestion
        "spi_lcd": {
            "start" : 0x80300000,
            "end"   : 0x80302000
        },
```

File: sdk/boards/sonata-prerelease.json [hunk 2]
```diff
@@ -52,37 +76,29 @@
...
-        "spi2": {
+        "spi_ethmac": {
             "start" : 0x80302000,
             "end"   : 0x80303000
         },
```
Comment (HU90m) [inline sdk/boards/sonata-prerelease.json:90]:
```suggestion
        "spi_ethmac": {
            "start" : 0x80301000,
            "end"   : 0x80303000
        },
```

File: sdk/boards/sonata-prerelease.json [hunk 3]
```diff
@@ -52,37 +76,29 @@
             "start" : 0x80201000,
             "end"   : 0x80201080
         },
-        "spi0": {
+        "spi_board": {
             "start" : 0x80300000,
             "end"   : 0x80301000
         },
```
Comment (HU90m) [inline sdk/boards/sonata-prerelease.json:82]:
```suggestion
        "spi_board": {
            "start" : 0x80302000,
            "end"   : 0x80301000
        },
```

**Review by HU90m — CHANGES_REQUESTED** [review]

File: sdk/include/platform/sunburst/platform-pwm.hh
Comment (HU90m) [inline sdk/include/platform/sunburst/platform-pwm.hh:1]: Could we hold off on pulling the PWM driver changes? I'm still not sure about them.

**Comment by AlexJones0** [comment]
Closed as superseded by #20.

**Pull request closed without merging**